js/main.js

```diff
@@ -31,105 +54,421 @@ document.addEventListener('DOMContentLoaded', function () {
+    lunchSelected = true;
+  }
+  if (dinnerCounter === 1 && !dinnerSelected) {
+    renderDinnerEntry();
+    dinnerSelected = true;
+  }
+}
+
+function renderBreakfastEntry() {
+
+  for (let i = 0; i < data.breakfastFavorites.length; i++) {
+    const $tr = document.createElement('tr');
+    const $td = document.createElement('td');
+    const $textContent = document.createTextNode(data.breakfastFavorites[i].title);
+    const $tdTwo = document.createElement('td');
+    const $buttonMealInfo = document.createElement('button');
+    const $buttonMealInfoText = document.createTextNode('MEAL INFO');
+    const $buttonMealNotes = document.createElement('button');
+    const $buttonMealNotesText = document.createTextNode('NOTES');
+    $breakfastTable.appendChild($tr);
+    $tr.appendChild($td);
+    $td.appendChild($textContent);
+    $tr.appendChild($tdTwo);
+    $tdTwo.appendChild($buttonMealInfo);
+    $buttonMealInfo.appendChild($buttonMealInfoText);
+    $tdTwo.appendChild($buttonMealNotes);
+    $buttonMealNotes.appendChild($buttonMealNotesText);
+  }
+  return $breakfastTable;
+
+}
+
+function renderLunchEntry() {
+  for (let i = 0; i < data.lunchFavorites.length; i++) {
+    const $tr = document.createElement('tr');
+    const $td = document.createElement('td');
+    const $textContent = document.createTextNode(data.lunchFavorites[i].title);
+    const $tdTwo = document.createElement('td');
+    const $buttonMealInfo = document.createElement('button');
+    const $buttonMealInfoText = document.createTextNode('MEAL INFO');
+    const $buttonMealNotes = document.createElement('button');
+    const $buttonMealNotesText = document.createTextNode('NOTES');
+    $lunchTable.appendChild($tr);
+    $tr.appendChild($td);
+    $td.appendChild($textContent);
+    $tr.appendChild($tdTwo);
+    $tdTwo.appendChild($buttonMealInfo);
+    $buttonMealInfo.appendChild($buttonMealInfoText);
+    $tdTwo.appendChild($buttonMealNotes);
+    $buttonMealNotes.appendChild($buttonMealNotesText);
+  }
+  return $lunchTable;
+}
+function renderDinnerEntry() {
+  for (let i = 0; i < data.dinnerFavorites.length; i++) {
+    const $tr = document.createElement('tr');
+    const $td = document.createElement('td');
+    const $textContent = document.createTextNode(data.dinnerFavorites[i].title);
+    const $tdTwo = document.createElement('td');
+    const $buttonMealInfo = document.createElement('button');
+    const $buttonMealInfoText = document.createTextNode('MEAL INFO');
+    const $buttonMealNotes = document.createElement('button');
+    const $buttonMealNotesText = document.createTextNode('NOTES');
+    $dinnerTable.appendChild($tr);
+    $tr.appendChild($td);
+    $td.appendChild($textContent);
+    $tr.appendChild($tdTwo);
+    $tdTwo.appendChild($buttonMealInfo);
+    $buttonMealInfo.appendChild($buttonMealInfoText);
+    $tdTwo.appendChild($buttonMealNotes);
+    $buttonMealNotes.appendChild($buttonMealNotesText);
+  }
+  return $dinnerTable;
+}
```

Comment:
Since the DOM tree for these are completely the same, it would make more sense to combine this into 1 one function. 

Rather than looping over the different arrays inside these functions, you should do the loop where you call the function, and pass in each object from the array into this new render function. 

Same thing goes for your appending to the DOM. Rather than appending to `$lunchTable` or `$dinnerTable` directly inside of the render function, you should be doing that where you are calling the render function, and return your `$tr` element.